resources/etcd_installation_binary.rb

```diff
@@ -76,24 +86,8 @@ def default_checksum
   end
 end
 
-def etcd_bin
-  "#{etcd_bin_prefix}/etcd"
-end
-
-def etcd_bin_prefix
-  '/usr/bin'
-end
-
-def etcdctl_bin
-  "#{etcdctl_bin_prefix}/etcdctl"
-end
-
-def etcdctl_bin_prefix
-  etcd_bin_prefix
-end
```

Comment:
These are now only declared in a single place, `libraries/helpers_service.rb`, rather than duplicated.